src/foam/u2/ModalHeader.js

```diff
@@ -79,8 +79,7 @@ foam.CLASS({
 
     this
       .addClass(this.myClass())
-      .start().addClass('title').add(this.title).end()
-      .start(this.CLOSE_MODAL).addClass('close').end();
```

Comment:
It is not a good idea remove this `Close` button, it will affect all the app which is using the `ModalHeader.js`